src/pages/AdminMessages.tsx

```
@@ -99,8 +219,99 @@ const AdminMessages: React.FC = () => {
+                  <h2 className="admin-section-title">Messages Internes</h2>
+                  <div className="space-y-3">
+                    {(data?.internalMessages || []).slice().reverse().map(m => (
+                      <div key={m.id} className="bg-black p-4 border border-pm-gold/10 rounded-lg">
+                        <div className="flex justify-between items-start">
+                          <div>
+                            <p className="text-sm text-pm-off-white/70">De <span className="font-semibold">{m.from?.name || 'Inconnu'}</span> ➜ {(m.to || []).filter(Boolean).map((t) => t?.name || '—').join(', ')}</p>
+                            <h3 className="text-lg font-bold text-pm-gold mt-1">{m.subject || '(Sans objet)'}</h3>
+                          </div>
+                          <p className="text-xs text-pm-off-white/60">{new Date(m.createdAt).toLocaleString('fr-FR')}</p>
+                        </div>
+                        <div className="mt-3 text-sm text-pm-off-white/90" dangerouslySetInnerHTML={{ __html: m.body || '' }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)